posthog/utils.py

```diff
@@ -1068,6 +1068,20 @@ def filters_override_requested_by_client(request: Request) -> Optional[dict]:
     return None
 
 
+def variables_override_requested_by_client(request: Request) -> Optional[dict[str, dict]]:
```

Comment:
NIT: there's a posthog/api/utils.py file that could be good for this logic?

Reply:
This is in the same file as `filters_override_requested_by_client` (directly above) - so thought this was best placed than spreading the funcs out